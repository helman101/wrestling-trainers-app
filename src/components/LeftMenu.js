import React from 'react';
import { Link } from 'react-router-dom';
import PropTypes from 'prop-types';

const LeftMenu = (props) => {
  const { user } = props;
  return (
    <aside>
      <div>
        <Link to="/">Home</Link>
        <Link to="/how_to">How to</Link>
        <Link to="/trainers">Trainers</Link>
        { user && <Link to="/my_appointment">My Appointments</Link> }
      </div>
    </aside>
  );
};

LeftMenu.propTypes = {
  user: PropTypes.instanceOf(Object).isRequired,
};

export default LeftMenu;
