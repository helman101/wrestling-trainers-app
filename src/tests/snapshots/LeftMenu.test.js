import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import LeftMenu from '../../components/LeftMenu';
 
describe('LeftMenu', () => {
  let component;
 
  beforeEach(() => {
    component = renderer.create(
      <MemoryRouter>
        <LeftMenu />
      </MemoryRouter>
    )
  });
 
  it('should render propertly', () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});
