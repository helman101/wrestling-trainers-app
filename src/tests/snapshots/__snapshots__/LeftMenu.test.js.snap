// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftMenu should render propertly 1`] = `
<aside
  className="leftMenu"
>
  <ul
    className="leftList mt5 ml25p"
  >
    <li>
      <a
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        href="/how_to"
        onClick={[Function]}
      >
        How to
      </a>
    </li>
    <li>
      <a
        href="/trainers"
        onClick={[Function]}
      >
        Trainers
      </a>
    </li>
  </ul>
</aside>
`;
